cmd/geth/main.go

```diff
@@ -430,8 +431,10 @@ func startNode(ctx *cli.Context, stack *node.Node, backend ethapi.Backend, isCon
 			binary.BigEndian.Uint64(utxoHeight), "txIndexHeight", binary.BigEndian.Uint64(txHeight))
 	}
 
-	var initHeight uint64
-	initHeight = 2585811 // Temp for testing, this should be part of chain config
+	var initHeight uint64 = 2585811
```

Comment:
Could we make this variable a global constant? 🤔

Reply:
sure!